web/hand_tool.js

```diff
@@ -79,27 +96,31 @@ var HandTool = {
         }
       }.bind(this));
     }
-  },
+  }
 
-  toggle: function handToolToggle() {
-    this.handTool.toggle();
-    SecondaryToolbar.close();
-  },
+  HandTool.prototype = {
```

Comment:
Also, based on a recent question (issue 7187): Would it make sense to add a simple getter, to be able to tell if the hand tool is active?
E.g. something like this:
```javascript
/**
 * @return {boolean}
 */
get active() {
  return !!this.handTool.active; // The !! is because grab_to_pan.js doesn't have a default value.
}
```